src/sql/src/session/vars.rs

```diff
@@ -656,6 +656,13 @@ const PERSIST_FAST_PATH_LIMIT: ServerVar<usize> = ServerVar {
     internal: true,
 };
 
+const TIMESTAMP_ORACLE_IMPL: ServerVar<TimestampOracleImpl> = ServerVar {
+    name: UncasedStr::new("timestamp_oracle"),
+    value: &TimestampOracleImpl::Catalog,
```

Comment:
Heh, I'd have used an `enable_postgres_timestamp_oracle` bool to make this more like the normal feature flags, but this works too :)

Reply:
it's not enough, no, and we need the commandline arg to, say, initialize the catalog oracle from the postgres oracle, in case we're switching back. And the idea was to add the commandline arg (and have cloud deal with adding the schema/database for it), and we'd have the LD flag for then enabling or disabling the oracle at will.

Also, and here I could be wrong and haven't set things up correctly: I think normally you wouldn't use `ALTER SYSTEM` but change it in LD.